dev/src/reference.ts

```diff
@@ -1879,6 +1920,34 @@ export class Query<T = firestore.DocumentData> implements firestore.Query<T> {
     });
   }
 
+  /**
+   * Internal get() method that returns all descendants of the current doc.
+   *
+   * @private
+   * @param {bytes=} transactionId A transaction ID.
+   */
+  _getAllDescendants(transactionId?: Uint8Array): Promise<DocumentReference[]> {
+    const docs: DocumentReference[] = [];
+
+    // Capture the error stack to preserve stack tracing across async calls.
+    const stack = Error().stack!;
+
+    return new Promise((resolve, reject) => {
+      this._stream(transactionId)
+        .on('error', err => {
+          reject(wrapError(err, stack));
+        })
+        .on('data', result => {
+          if (result.document) {
+            docs.push(result.document);
+          }
+        })
+        .on('end', () => {
+          resolve(docs);
+        });
+    });
+  }
```

Comment:
Can we not use the existing method and return DocumentSnapshots with empty data() (similar to what `Query.select(FieldPath.documentId())` already does?)